panels/templates/attractions/events.html

```diff
@@ -267,52 +266,49 @@ <h4 class="modal-title">
           <span class="bling-icon"></span> <span class="signup-event-label"></span>
         </h4>
       </div>
-      <div class="modal-body">
```

Comment:
I look forward to seeing how this is now styled.  I figured you could do a much better job than me with making this modal look good - I spent an embarrassing amount of time getting it to look as decent as it did, but I was never really happy with it.

Reply:
Visually, it looks _almost_ exactly the same. But it does the right thing on mobile screens (uses the full width vs a fixed width/fixed margin).

One big change I made was to move the `modal-footer` inside the `form` element. Additionally, instead of hooking the `button.click` event, I am hooking the `form.submit` event. I call `event.preventDefault()`, so we don't actually POST the form – we still use AJAX to post the data the way we did before.

Moving the `model-footer` inside the `form` element and hooking `form.submit` instead of `button.click` has several cool benefits:
* It allows you to hit the enter key in a text field to submit the data instead of requiring a mouse click on the button, or tabbing the button into focus
* It gets HTML5 form validation by the browser for free. The browser intercepts the `form.submit` event before it gets to our handler, and does client-side form validation (like "Fill out this field", see screenshot below).

Also, I removed the rounded corners from the middle fields.

# Old: Rounded Corners on Internal Fields
<img width="628" alt="screen shot 2017-12-28 at 6 53 35 am" src="https://user-images.githubusercontent.com/2592431/34410134-1ca2a6d8-eb9c-11e7-9ae0-32d301b2db5d.png">

# New: Sharp Corners on Internal Fields
<img width="626" alt="screen shot 2017-12-28 at 7 04 09 am" src="https://user-images.githubusercontent.com/2592431/34410326-5cbde5b0-eb9d-11e7-9b75-82a1ab8b9a33.png">